src/main/java/com/hazelcast/kubernetes/HazelcastKubernetesDiscoveryStrategy.java

```diff
@@ -99,6 +116,36 @@ public void start() {
         endpointResolver.start();
     }
 
+    @Override
+    public Map<String, Object> discoverLocalMetadata() {
+        if (memberMetadata.isEmpty()) {
+            memberMetadata.put(PartitionGroupMetaData.PARTITION_GROUP_ZONE, discoverZone());
+        }
+        return memberMetadata;
+    }
+
+    private String discoverZone() {
+        try {
+            String podName = System.getenv("POD_NAME");
+            if (podName == null) {
+                podName = System.getenv("HOSTNAME");
+            }
```

Comment:
Are we going to advise people to set HOSTNAME or POD_NAME? or they are for people who wants to be on the safe side.

Reply:
Yeah, I'd recommend to just use HOSTNAME (I didn't experience any issues with that and I tried it in three different environments: locally, Azure, GKE). But I'd mention also that if you change hostname anywhere, then please define POD_NAME. I'd also add POD_NAME to the Helm Chart, since it's zero effort.

It's kind-of, pragmatic, convention-based approach. Like Spring Boot, it just works if you don't define anything. But if you do some complex stuff, you may need to define something extra.

Reply:
I think we need to list in which case HOSTNAME or POD_NAME is needed and tell the world it just works except those cases.

Reply:
ok, since it's about the documentation part, let's move this discussion to #123